lib/atom-types.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+import type { Range } from 'atom'
+
+type Buffer = {
+    characterIndexForPosition: (position: Range) => number,
+    positionForCharacterIndex: (index: number) => any,
+    getTextInRange: (range: Range) => string,
+}
+
+type Point = {
+    toArray: () => [number, number]
+}
+
+export type TextEditor = {
```

Comment:
I was wrong here. `require('atom').TextEditor` does exist, so these should be define in the module.

`require('atom') !== global.atom`